odmlui/DocumentRegistry.py

```diff
@@ -19,7 +20,8 @@ def add(self, doc):
         """
         add a document to the registry and return its unique id
         """
-        id = hash(doc)
+        # odML.Document is unhashable. Converting to string before hashing
+        id = hash(str(doc))
```

Comment:
if Document is unhashable and we need it to be, it should implement `__hash__()`, right?